### PR TITLE
[#IP-359] Update Node to 14.16.0

### PR DESCRIPTION
**Pull request by balanza**
Update Node to 14.16.0

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -251,12 +252,18 @@ export function withSpid({
...
+            .filter((e): e is string => typeof e === "string")
             .chain(authLevel =>
               lookup(authLevel, SPID_LEVELS).map(_ => authLevel)
             )
             .chain(
-              fromPredicate(authLevel =>
-                appConfig.spidLevelsWhitelist.includes(authLevel)
+              fromPredicate(
+                authLevel =>
+                  t.keyof(SPID_LEVELS).is(authLevel) &&
```
Comment (BurnedMarshal) [inline src/index.ts:265]: If we check here the value of `authLevel` with `t.keyof(SPID_LEVELS).is(authLevel)` maybe we can remove the chain/lookup before or remove the `is` statement adding a casting like
```typescript
.chain(authLevel =>
  lookup(authLevel, SPID_LEVELS).map(_ => authLevel as keyof SPID_LEVELS)
)
```

File: yarn.lock [hunk 2]
```diff
@@ -1,5865 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (BurnedMarshal) [inline yarn.lock:1]: This file was totally removed. Is this change wanted?

**Review by BurnedMarshal — CHANGES_REQUESTED** [review]

File: yarn.lock (hunk 2 above)
Comment (balanza) [reply yarn.lock:1]: nope, it's a mistake, I'll fix it

Comment (balanza) [reply yarn.lock:1]: fixed in [`db88a8e` (#88)](https://github.com/pagopa/io-spid-commons/pull/88/commits/db88a8e958a2a08382a97d2c264928c93c868ba4)

File: src/index.ts (hunk 1 above)
Comment (balanza) [reply src/index.ts:265]: Nice point, I did this: https://github.com/pagopa/io-spid-commons/pull/88/commits/2a9c5715e069e18e760868e5cda43ea3840e9ac4

File: yarn.lock (hunk 2 above)
Comment (balanza) [reply yarn.lock]: fixed in https://github.com/pagopa/io-spid-commons/pull/88/commits/cb562179b495421207f954da2d181d647857fee9

File: src/index.ts (hunk 1 above)
Comment (balanza) [reply src/index.ts]: Nice point, I did this: https://github.com/pagopa/io-spid-commons/pull/88/commits/b416411d3e3db6a64a4d1805d33b8e88548be1ab

**Review by BurnedMarshal — APPROVED** [review]
LGTM

**Pull request merged**